### PR TITLE
feat(web): implement lepton docs page under /lep-docs

### PR DESCRIPTION
**Pull request by yangjunhan**

### CODE REVIEW

File: leptonai/photon/photon.py [hunk 1]
```diff
@@ -830,6 +835,21 @@ def queue_length():
                 num_tasks -= 1
             return num_tasks
 
+        file_dir = Path(__file__).parent
+        ui_path = (file_dir / "../ui").resolve()
```
Comment (bddppq) [inline leptonai/photon/photon.py:839]: nit: this might not work on windows, better to use `path.parent`

File: leptonai/ui/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/lep-docs/leptonai.svg" />
```
Comment (bddppq) [inline leptonai/ui/index.html:5]: we have one in `assets/logo.svg`. maybe consolidate these two?

**Review by bddppq — APPROVED** [review]
Overall LGTM

File: leptonai/ui/index.html (hunk 2 above)
Comment (yangjunhan) [reply leptonai/ui/index.html:5]: `leptonai.svg` is a svg consists of a logo as well as an inline text, so it doesn't seem to be able to replace the existing `logo.svg`

**Pull request merged**